include/picomamba/picomamba_config.hpp

```diff
@@ -3,7 +3,7 @@
 #define PICOMAMBA_PICOMAMBA_CONFIG_HPP
 
 #define PICOMAMBA_VERSION_PATCH 0
-#define PICOMAMBA_VERSION_MINOR 2
+#define PICOMAMBA_VERSION_MINOR 3
 #define PICOMAMBA_VERSION_MAJOR 0
```

Comment:
Nitpicking: a good practice is to have a dedicated commit used for tagging a release, where you change the version numbers only (and any information related to the release only).